helm/cluster-cloud-director/values.schema.json

```
@@ -183,7 +183,23 @@
             "type": "string"
         },
         "kubectlImage": {
-            "type": "object"
+            "type": "object",
+            "title": "kubectl Image",
```

Let's please use sentence case

```suggestion
            "title": "Kubectl image",
```